lxd/patches.go

```diff
@@ -3444,6 +3445,30 @@ func patchClusteringDropDatabaseRole(name string, d *Daemon) error {
 	})
 }
 
+// patchNetworkCearBridgeVolatileHwaddr removes the unsupported `volatile.bridge.hwaddr` config key from networks.
+func patchNetworkCearBridgeVolatileHwaddr(name string, d *Daemon) error {
+	// Get the list of networks.
+	networks, err := d.cluster.GetNetworks()
+	if err != nil {
+		return errors.Wrapf(err, "Failed loading networks for network_clear_bridge_volatile_hwaddr patch")
+	}
+
+	for _, networkName := range networks {
+		_, net, err := d.cluster.GetNetworkInAnyState(networkName)
+		if err != nil {
+			return errors.Wrapf(err, "Failed loading network %q for network_clear_bridge_volatile_hwaddr patch", networkName)
+		}
+
+		if net.Config["volatile.bridge.hwaddr"] != "" {
+			delete(net.Config, "volatile.bridge.hwaddr")
+			err = d.cluster.UpdateNetwork(net.Name, net.Description, net.Config)
+			return errors.Wrapf(err, "Failed updating network %q for network_clear_bridge_volatile_hwaddr patch", networkName)
```

Comment:
Missing an if statement here. As it stands, if you have multiple networks that need fixing, only the first will be processed as this return will hit.

Reply:
@stgraber good spot, fixed.